msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -134,13 +134,13 @@ With post-meeting app experience, you can view the results of the meeting, such
 
 The following image displays the **Contoso** tab with results of poll and feedback received from meeting attendees:
 
-# [Desktop](#tab/desktop)
+# [Desktop](#tab/desktop3)
 
-:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="Contoso tab with results.":::
+:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="The screenshot is an example that shows the Contoso tab with results in desktop.":::
 
-# [Mobile](#tab/mobile)
+# [Mobile](#tab/mobile3)
 
-:::image type="content" source="~/assets/images/apps-in-meetings/mobilepremeeting.png" alt-text="Post meeting app experience.":::
+:::image type="content" source="~/assets/images/apps-in-meetings/mobilepremeeting.png" alt-text="The screenshot is an example that shows the Post meeting app experience in mobile.":::
```

Comment:
Name conflict here too? Also, I can't seem to easily tell the difference between the 2 screenshots - mobilepremeeting and mobilepostmeeting. Please check once.

Reply:
resolved